client/my-sites/checkout/checkout/new-card-form.jsx

```diff
@@ -1,44 +1,48 @@
+import { isEmpty } from 'lodash';
+
 /**
  * External dependencies
  */
-import { isEmpty } from 'lodash';
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import PropTypes from 'prop-types';
+
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
-var CreditCardFormFields = require( 'components/credit-card-form-fields' ),
-	upgradesActions = require( 'lib/upgrades/actions' );
+import CreditCardFormFields from 'components/credit-card-form-fields';
+import upgradesActions from 'lib/upgrades/actions';
 
-module.exports = React.createClass( {
-	displayName: 'NewCardForm',
+class NewCardForm extends Component {
+	static displayName = 'NewCardForm';
 
-	propTypes: {
-		countriesList: React.PropTypes.object.isRequired,
-		hasStoredCards: React.PropTypes.bool.isRequired,
-		transaction: React.PropTypes.object.isRequired
-	},
+	static propTypes = {
+		countriesList: PropTypes.object.isRequired,
+		hasStoredCards: PropTypes.bool.isRequired,
+		transaction: PropTypes.object.isRequired
+	};
 
-	isFieldInvalid: function( fieldName ) {
+	isFieldInvalid = fieldName => {
 		return ! isEmpty( this.props.transaction.errors[ fieldName ] );
-	},
+	};
 
-	render: function() {
+	render() {
 		let classes = classNames( 'all-fields-required', { 'has-saved-cards': this.props.hasStoredCards } );
```

Comment:
Not going to end the world, but since we're not reassigning `classes` we could declare using `const`